python-avocado.spec

```diff
@@ -188,6 +188,14 @@ popd
 %{__install} -d -m 0755 %{buildroot}%{_sharedstatedir}/avocado/data
 %{__install} -d -m 0755 %{buildroot}%{_libexecdir}/avocado
 %{__install} -m 0755 libexec/avocado* %{buildroot}%{_libexecdir}/avocado
+%{__install} -d -m 0755 %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/gdb-prerun-scripts %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/plugins %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/tests %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/wrappers %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/yaml_to_mux %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/yaml_to_mux_loader %{buildroot}%{_datadir}/avocado
+%{__cp} -r examples/varianter_pict %{buildroot}%{_datadir}/avocado
```

Comment:
can't we simply do `%{__cp} -r examples %{buildroot}%{_datadir}/avocado/examples`?

Reply:
I skipped *all* examples, because the testplan directory doesn't make sense without the avocado-run-testplan, and I don't think we should be shipping it too.

Reply:
I see, I missed you're not copying everything. Hmm, how about `cp` followed by `rm`? Or is it possible to use advanced exprs like `^` in `cp`?